compute-vars/small.tfvars

```diff
@@ -0,0 +1,7 @@
+## Small Configuration Template
+
+bastion                     = {memory      = "16",   processors  = "0.5",   "count"   = 1}
+bootstrap                   = {memory      = "16",   processors  = "0.5",  "count"   = 1}
+master                      = {memory      = "16",   processors  = "0.5",  "count"   = 3}
```

Comment:
Only worry is that 16GB masters may fail the deployment with OOM issues. So it is of no use to give this flavor at all. We have experienced this on Power systems. @bpradipt @arcprabh

Reply:
That is right @yussufsh . Please set 32GB for the rhcos nodes for now.
For 4.8 onwards with RHEL 8.4, if the OOM issue is not recreated with 16GB, then we can consider updating it to 16.

Reply:
Updated the file with the changes